keyboards/sneakbox/aliceclone/aliceclone.h

```diff
@@ -19,7 +19,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #include "quantum.h"
 
-#define LAYOUT_all( \
+#define LAYOUT_default LAYOUT_alice
```

Comment:
I'm actively opposed to the change to the name here. 

If it's going to be changed, at all, backwards compatibility should be maintained. 

```suggestion
#define LAYOUT_all LAYOUT_alice
```

Reply:
Change made.

Is there a way you would prefer this is done?